src/main/java/org/reactome/server/service/controller/citation/CitationController.java

```diff
@@ -0,0 +1,294 @@
+package org.reactome.server.service.controller.citation;
```

Comment:
The formatting/alignment looks a bit strange in some if clauses throughout the class. Not sure if git messes here or if it misaligned. Could you check that please ?